frontend/pulse3d/src/components/basicWidgets/UsageWidgetFull.js

```diff
@@ -0,0 +1,51 @@
+import CircularProgressWithLabel from "./CircularProgressWithLabel";
+export default function UsageWidget({
+  metricName,
+  limitUsage,
+  actualUsage,
+  subscriptionName,
+  subscriptionEndDate,
+  labeltextcolor,
+  daysOfPlanLeft,
+}) {
+  return (
+    <>
+      <div id="container">
+        <h1>{subscriptionName} Plan</h1>
+        <p>{`Plan Expires on ${subscriptionEndDate}`}</p>
+        <p>
+          {daysOfPlanLeft >= 0
+            ? `${daysOfPlanLeft} days of plan left`
+            : `${daysOfPlanLeft * -1} days expired`}
+        </p>
+        <p>{`${actualUsage} out of ${limitUsage} ${metricName} used`}</p>
+        <CircularProgressWithLabel
+          value={(actualUsage / limitUsage) * 100 > 100 ? 100 : parseInt((actualUsage / limitUsage) * 100)}
+          labeltextcolor={labeltextcolor}
+        />
+        <p id="smallDescription">
+          Each upload comes with one free initial analysis and one free re-analysis. All subsequent analyses
```

Comment:
I thought the first analysis was counted and the first re-analysis wasn't? I ofc wasn't at the meeting for this, did we change it to a free analysis and re-analysis?

Reply:
Yeah, only the first re-analysis is free. We should change this